src/LuaTransformer.ts

```diff
@@ -3741,9 +3741,9 @@ export class LuaTransformer {
                 .replace(/\\/g, ".");
         }
         return filePath
+            .replace(/\.json$/, "")
```

Comment:
It should be before windows part as well. Or, maybe, just make windows part replace backslashes with slashes to make it cleaner?